### PR TITLE
A few of the initial classes

### PR DESCRIPTION
**Pull request by travisreed-wf**

### CODE REVIEW

File: src/game/game.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from google.appengine.ext import ndb
+from google.appengine.ext.ndb import polymodel
+
+
+class Game(polymodel.PolyModel):
+    date = ndb.DateTimeProperty(auto_now_add=True)
+    players = ndb.KeyProperty("User", repeated=True)
+    point_data = ndb.StructuredProperty(PointData, repeated=True)
+    winners = ndb.KeyProperty("User", repeated=True)
```
Comment (josephmcgovern-wf) [inline src/game/game.py]: kind=?

Comment (josephmcgovern-wf) [inline src/game/game.py]: To me it would make sense to relay the type in this variable name:
`player_keys`
`winner_keys`

File: src/game/game.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class PointData(ndb.StructuredProperty):
+    player = ndb.KeyProperty("User")
+    points = ndb.IntegerProperty()
```
Comment (josephmcgovern-wf) [inline src/game/game.py]: How do you plan on implementing this? What is `PointData` supposed to represent? I'm not a big fan of the name tbh

File: src/game/pool_game.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+from google.appengine.ext import ndb
+
+from src.game.game import Game
+
+class PoolGame(Game):
+    final_state = ndb.StructuredProperty(PlayerPoolBallsState, repeated=True)
+
+    @classmethod
+    def create(cls, final_state):
```
Comment (josephmcgovern-wf) [inline src/game/pool_game.py]: I think this should be `add`

File: src/game/pool_game.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        pass  # TODO
+
+
+class EightBallGame(PoolGame):
+    pass
+
+    def calculate_points(self):
+        pass  # TODO
+
+class PlayerPoolBallsState(ndb.StructuredProperty):
```
Comment (josephmcgovern-wf) [inline src/game/pool_game.py]: `TableState` or `FinalTableState`

File: src/game/game.py (hunk 1 above)
Comment (travisreed-wf) [inline src/game/game.py]: I agree

File: src/game/game.py (hunk 2 above)
Comment (travisreed-wf) [inline src/game/game.py]: yeah, we can talk this over when you get online

File: src/game/pool_game.py (hunk 3 above)
Comment (travisreed-wf) [inline src/game/pool_game.py]: sure

File: src/game/pool_game.py (hunk 4 above)
Comment (travisreed-wf) [inline src/game/pool_game.py]: this is the state for a single player though

File: src/user/player.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
+import flask
```
Comment (travisreed-wf) [inline src/user/player.py]: rename file to be user.py

**Pull request merged**